dev/TeachingTip/TeachingTip.cpp

```diff
@@ -1235,6 +1278,21 @@ winrt::TeachingTipPlacementMode TeachingTip::DetermineEffectivePlacement()
             bool leftCenterAvailable = true;
             bool leftTopAvailable = true;
             bool leftBottomAvailable = true;
+            auto screenSize = winrt::Size({ 0.0f, 0.0f });
+            if (!shouldConstrainToRootBounds)
+            {
+                if (m_useTestScreenBounds)
+                {
+                    screenSize = winrt::Size(m_testScreenBounds.Width, m_testScreenBounds.Height);
+                }
+                else
+                {
+                    auto displayInfo = winrt::DisplayInformation::GetForCurrentView();
+                    auto rawPixelsPerViewPixel = displayInfo.RawPixelsPerViewPixel();
```

Comment:
Does this API have an islands-friendly equivalent yet? (it may not)

Reply:
No

---
In reply to: [268776198](https://github.com/Microsoft/microsoft-ui-xaml/pull/480#discussion_r268776198) [](ancestors = 268776198)